deploy/olm-catalog/host-operator/host-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
```

Comment:
@sbryzak This file shouldn't be here - we renamed the operator to `toolchain-host-operator` a long time ago. It looks like you generated the stuff against an outdated version of the host-operator repo